alarms/dojot.auth.WrongPassword.xml

```diff
@@ -0,0 +1,12 @@
+<alarm-metamodel>
```

Comment:
I think this should be merged with `AuthenticationError`

Reply:
Ok ... in the auth code both names are used, but I will remove this one.